src/net.cpp

```diff
@@ -60,6 +63,7 @@
 
 namespace {
     const int MAX_OUTBOUND_CONNECTIONS = 8;
+    const int MAX_FEELER_CONNECTIONS = 1;
```

Comment:
It is used indirectly in https://github.com/bitcoin/bitcoin/pull/8282/files/4b4517ce0340c813553f17d1e7710129542d58bd#diff-9a82240fe7dfe86564178691cc57f2f1R1672